bin/main.rb

```diff
@@ -1,7 +1,12 @@
 #!/usr/bin/env ruby
 
-class Game
+require '../lib/player.rb'
+require '../lib/board.rb'
```

Comment:
- [ ] [Optional] Allways use `require_relative` for internal dependencies (like files you create) and `require` for external dependencies (likes gems or packages, more on this in the Rails module) - [link with details](https://github.com/rubocop-hq/ruby-style-guide#use-require_relative-whenever-possible)